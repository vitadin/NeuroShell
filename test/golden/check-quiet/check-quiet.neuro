%% Test check command quiet mode
\set[_echo_commands="true"]
%% This test verifies that \check[quiet=true] doesn't produce console output
%% but still sets the result variables
\check[quiet=true]
\echo Status: ${_check_status}
\echo Total services: ${_check_total_services}
\echo Failed count: ${_check_failed_count}