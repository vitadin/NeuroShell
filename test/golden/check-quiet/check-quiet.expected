Setting _echo_commands = true
%%> \check[quiet=true]
%%> \echo Status: ${_check_status}
Status: success
%%> \echo Total services: ${_check_total_services}
Total services: 15
%%> \echo Failed count: ${_check_failed_count}
Failed count: 0